### PR TITLE
Add new hook displayFullWidthTop

### PR DESCRIPTION
**Pull request by iqit-commerce**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | For many merchant current displayTop hook maybe to narrow for displaying main menu and they have no option to display full width menu without template modification |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | This PR requires a module hooked on displayFullWidthTop. |

### CODE REVIEW

**Comment by xBorderie** [comment]
Checks have failed. Could you rebase your PR in order to restart the tests?

**Comment by firstred** [comment]
Almost same PR as https://github.com/PrestaShop/PrestaShop/pull/6500

File: install-dev/data/xml/hook.xml [hunk 1]
```diff
@@ -57,6 +57,9 @@
     <hook id="displayTop">
       <name>displayTop</name><title>Top of pages</title><description>This hook displays additional elements at the top of your pages</description>
     </hook>
+    <hook id="displayFullWidthTop">
+      <name>displayFullWidthTop</name><title>Top of pages without width limitation.</title><description>This hook displays additional elements at the top of your pages without width limitation.</description>
+    </hook>
```
Comment (aleeks) [inline install-dev/data/xml/hook.xml]: Hello @iqit-commerce 

Missing sql update into 1.7.0.0.sql nop ?

INSERT INTO PREFIX_hook(name,title,description,position) VALUES ('displayFullWidthTop', 'Top of pages without width limitation.', 'This hook displays additional elements at the top of your pages without width limitation.', '1');

**Review by aleeks — CHANGES_REQUESTED** [review]

**Comment by Shudrum** [comment]
Hi, thank you for your contribution.

I close as the #6691 is more useful for the same idea.

**Pull request closed without merging**